src/LiberisLabs.CompaniesHouse.IntegrationTests/Tests/CompanyFilingHistoryTests/CompanyFilingHistoryTestsInvalid.cs

```diff
@@ -0,0 +1,27 @@
+﻿using NUnit.Framework;
+
+namespace LiberisLabs.CompaniesHouse.IntegrationTests.Tests.CompanyFilingHistoryTests
+{
+    [TestFixture]
+    public class CompanyFilingHistoryTestsInvalid : CompanyFilingHistoryTestBase
+    {
+        private const string InvalidCompanyNumber = "ABC00000";
+
+        [SetUp]
+        protected override void When()
+        {
+            WhenRetrievingAnInvalidCompanyProfile();
+        }
+
+        [Test]
+        public void ThenTheProfileIsNotReturned()
+        {
+            Assert.That(_result.Data.Items, Is.Null);
```

Comment:
Shouldn't `_result.Data` be null? as having the `_result.Data.Items` null makes that a bit ambiguous as whats the difference between a empty list and a null list?

if we could change this to just pass up the `_result.Data` back as null that be great! keeps it consistent with the Company Profile client too - https://github.com/LiberisLabs/CompaniesHouse.NET/blob/master/src/LiberisLabs.CompaniesHouse.IntegrationTests/Tests/CompanyProfileTests/CompanyProfileTestsInvalid.cs#L19